meilisearch-http/src/routes/indexes/search.rs

```diff
@@ -81,6 +79,30 @@ impl From<SearchQueryGet> for SearchQuery {
     }
 }
 
+/// Transform the sort query parameter into something that matches the post expected format.
```

Comment:
I'm not sure to understand why we need to fix the sort parameter

Reply:
Because when we get the sort parameters on the `post` route, we get something looking like that:
```
["price:asc", "date:desc"]
```

While on the `get` route, we get something like that:

```
"price:asc,date:desc"
```

What we were doing until now was a split on the `,` and the parse each thing independently.
Sadly, since the `_geoPoint(x, y)` contains a `,` we can't just do that and hope it works.
So one solution would be to write a full parser, but that would be a lot of code.
The solution I choose is to still split on the `,` but when I encounter `_geoPoint(...` I join all parameters until I find something ending by `):asc` or `):desc`.
It's a bit hacky, but it should work well for our use case.